src/components/playback/mediasession.js

```diff
@@ -49,121 +30,96 @@ define(['playbackManager', 'nowPlayingHelper', 'events', 'connectionManager'], f
         if (!item) {
             hideMediaControls();
             return;
         }
 
-        if (eventName == 'init') { // transform "init" event into "timeupdate" to restraint update rate
+        if (eventName === 'init') { // transform "init" event into "timeupdate" to restraint update rate
             eventName = 'timeupdate';
         }
 
-        var isVideo = item.MediaType === 'Video';
-        var isLocalPlayer = player.isLocalPlayer || false;
+        const isVideo = item.MediaType === 'Video';
+        const isLocalPlayer = player.isLocalPlayer || false;
 
         // Local players do their own notifications
         if (isLocalPlayer && isVideo) {
             return;
         }
 
-        var playState = state.PlayState || {};
-        var parts = nowPlayingHelper.getNowPlayingNames(item);
-        var artist = parts[parts.length - 1].text;
-        var title = parts.length === 1 ? '' : parts[0].text;
-        var albumArtist;
-
-        if (item.AlbumArtists && item.AlbumArtists[0]) {
-            albumArtist = item.AlbumArtists[0].Name;
-        }
+        const playState = state.PlayState || {};
+        const parts = nowPlayingHelper.getNowPlayingNames(item);
+        const artist = parts[parts.length - 1].text;
+        const title = parts.length === 1 ? '' : parts[0].text;
 
-        var album = item.Album || '';
-        var itemId = item.Id;
+        const album = item.Album || '';
+        const itemId = item.Id;
 
         // Convert to ms
-        var duration = parseInt(item.RunTimeTicks ? (item.RunTimeTicks / 10000) : 0);
-        var currentTime = parseInt(playState.PositionTicks ? (playState.PositionTicks / 10000) : 0);
+        const duration = parseInt(item.RunTimeTicks ? (item.RunTimeTicks / 10000) : 0);
+        const currentTime = parseInt(playState.PositionTicks ? (playState.PositionTicks / 10000) : 0);
 
-        var isPaused = playState.IsPaused || false;
-        var canSeek = playState.CanSeek || false;
+        const isPaused = playState.IsPaused || false;
+        const canSeek = playState.CanSeek || false;
 
-        if (navigator.mediaSession) {
+        if ('mediaSession' in navigator) {
             navigator.mediaSession.metadata = new MediaMetadata({
                 title: title,
                 artist: artist,
                 album: album,
-                artwork: getImageUrls(item),
-                albumArtist: albumArtist,
-                currentTime: currentTime,
-                duration: duration,
-                paused: isPaused,
-                itemId: itemId,
-                mediaType: item.MediaType
+                artwork: getImageUrls(item)
             });
         } else {
-            var imageUrl = [];
-            pushImageUrl(item, {maxHeight: 400}, imageUrl);
-
-            if (imageUrl.length) {
-                imageUrl = imageUrl[0].src;
-            } else {
-                imageUrl = null;
+            let itemImageUrl = seriesImageUrl(item) || imageUrl(item);
```

Comment:
why `maxHeight` is gone?

Reply:
This PR originated from @dkanada noticing that artwork on Android was VERY low resolution, most notably the album art on the lockscreen.

Since I couldn't find any recommendation from Google on which resolution to use, I removed the limit.

Reply:
I think it makes sense to leave it at full res, so each device decides better how to fit it. It's only one image as well.

Reply:
Maybe we should limit it to FullHD or smth? I'm imagining the 8K artwork being transferred to the phone over not-so-fast 3G...

Reply:
Apparently, the recommended size for Spotify is 3000x3000.

However, there is not resolution limit, the only real limit being a size under 4MB.

I think limiting the max height to 3000px should work well enough.